### PR TITLE
B23 login functionality is done

### PR DESCRIPTION
**Pull request by ahmetYilmaz88**
this is a pull request opened after setting up branch protection rule, it should not merger without any review

### CODE REVIEW

**Comment by Feruzka1706** [comment]
reviewing

**Comment by ahmetYilmaz88** [comment]
please review

File: src/HelloGroup2.java [hunk 1]
```diff
@@ -1,10 +1,5 @@
 public class HelloGroup2 {
 
-    // Good morning Group2
-    // Group2 work has started
-    // We have good collaboration
```
Comment (Feruzka1706) [inline src/HelloGroup2.java:5]: reviewing

**Review by ahmadsam33r — COMMENTED** [review]
great

**Review by ahmetYilmaz88 — COMMENTED** [review]
great

**Comment by Feruzka1706** [comment]
how can we approve this one?

**Review by Feruzka1706 — APPROVED** [review]

**Review by ahmadsam33r — APPROVED** [review]

**Pull request merged**